### DISCUSSION TITLE
How to load .model and .optimizer file for load pytorch model?

### DISCUSSION THREAD

**Discussion by LxyoryxL** [Trusted-AI/adversarial-robustness-toolbox#1308] [Q&A]
![image](https://user-images.githubusercontent.com/35552946/132969956-8a24a675-3906-4a2f-8109-390025fe9af2.png)
The `.model` and `.Optimizer` files above are the ones I saved with `pytorch_model.save()` functions,but I don't know how to load them again

**Comment by beat-buesser** [comment]
Hi @LxyoryxL Have you tried loading them with `torch.load`?

**Reply by telemorne** [reply]
I used this and I got an error 
![image](https://user-images.githubusercontent.com/54008081/200076931-e7fc392d-c08c-4d08-8c53-c99403ebd5e7.png)

**Reply by telemorne** [reply]
how should the two files be passed to the load function?

**Reply by beat-buesser** [reply]
I think the correct way to make a prediction with a PyTorch-model is `classifier(x_test)`.
